src/GeositeFramework/js/Map.js

```diff
@@ -203,11 +213,13 @@ require(['use!Geosite',
         // the app entirely.  The map does, in fact, load and its loaded property is
         // set.  I put in this hack to check up on the event a little while after
         // it was created and manually raise the event if the library didn't do it.
+        /*
```

Comment:
This reads like an IE8 specific hack.  Could you add a TODO comment to delete once we confirm that it's not useful anymore?